mubench.reviewsite/tests/StoreFindingsTest.php

```diff
@@ -172,4 +173,45 @@ function test_get_misuse_ex1(){
         self::assertEquals($expected_misuse, $misuse);
     }
 
+    function test_delete_snippet()
+    {
+        $uploader = new FindingsUploader($this->db, $this->logger);
+
+        $data = json_decode(json_encode($this->request_body));
+        $uploader->processData("ex2", $data);
+
+        $snippet_uploader = new SnippetUploader($this->db, $this->logger);
+        $snippet_uploader->deleteSnippet("1");
+
+        $detector = $this->db->getOrCreateDetector("-d-");
+        $runs = $this->db->getRuns($detector, "ex2");
+
+        $expected_run = [
```

Comment:
Please only assert that the snippet is gone. The other values are not relevant to this test, hence, it should not fail when they change for whatever reason.